robot/web/web_c.go

```diff
@@ -84,7 +82,7 @@ func (svc *webService) streamInitialized() bool {
 
 func (svc *webService) addNewStreams(ctx context.Context) error {
 	if !svc.streamInitialized() {
-		return nil
+		return errors.New("stream server not initialized")
 	}
```

Comment:
Why are we returning an error now?

Reply:
Good call. Moved to warning log and returning nil.